index.js

```diff
@@ -4,6 +4,29 @@ const bodyParserMW = require('koa-bodyparser');
 
 const router = require('./lib/routes');
 
+const errorMiddleware = async (ctx, next) => {
+  try {
+    await next()
+  } catch (err) {
+    console.error(err);
+
+    ctx.status = err.status || 500;
+    ctx.set('Content-Type', 'application/json');
+
+    return {
+      success: false,
+      error: err.message,
+      data: null,
+    };
+  } finally {
+    return {
+      success: true,
+      error: null,
+      data: ctx.response.body,
+    }
+  }
+};
+
```

Comment:
Лучше вынести в отдельный файл миддлвар данный функционал
```suggestion
const errorMiddleware = async (ctx, next) => {
  try {
    await next()
  } catch (err) {
    console.error(err);

    ctx.status = err.status || 500;
    ctx.set('Content-Type', 'application/json');

    return {
      success: false,
      error: err.message,
      data: null,
    };
  } finally {
    return {
      success: true,
      error: null,
      data: ctx.response.body,
    }
  }
};

```